package.json

```diff
@@ -61,6 +61,7 @@
     "reset:install": "yarn install",
     "restart": "yarn reset && yarn start",
     "start": "yarn storybook dev -p ${npm_package_storybook_port}",
+    "start:local": "yarn storybook dev -p 7777",
```

Comment:
This should not be included. You can change the port number in your package.json for an individual project with `"sbport": 6099,` You can update that as you like, and commit to your project repo.